src/Arcus.Observability.Telemetry.Core/MessagePrefixes.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Arcus.Observability.Telemetry.Core
+{
+    public class MessagePrefixes
```

Comment:
Static class?

Reply:
It's for a more fluent approach of using them but open for suggestions